client/public/index.html

```diff
@@ -14,7 +14,8 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />-->
+    <link rel="“manifest”" href="/src/manifest.json" />
```

Comment:
This is the file that react produces by default, you edited it, now the link to the manifest.json is broken, not that it affects us but you broke something that you could have left, revert this.